spec/lib/job_posting_spec.rb

```diff
@@ -6,6 +6,7 @@
     {
       '@type' => 'JobPosting',
       'title' => 'Teacher of English',
+      'jobRole' => [I18n.t('jobs.job_role_options.nqt_suitable'), I18n.t('jobs.job_role_options.sen_specialist')],
       'salary' => 'Lots of money',
```

Comment:
As per comm's comment in the last show and tell, and while you're working in this part of the app, can you go ahead and change this copy here to something that doesn't worry them, please?  Maybe just some lorem.